File: Course_01/C++ Language/EstruturaSequencial/retangulo.cpp
#include <iostream>
#include <math.h>
#include <iomanip>

using namespace std;

int main()
{
    double diagonal, perimetro, area,  base, altura;

    cout << "Base do retangulo: ";
    cin >> base;
    cout << "Altura do retangulo: ";
    cin >> altura;

    area = base * altura;
    perimetro = 2 * base + 2 * altura;
    diagonal = sqrt(pow(base, 2) + pow(altura, 2));

    cout << fixed << setprecision(4);
    cout << "AREA = " << area << endl;
    cout << "PERIMETRO = " << perimetro << endl;
    cout << "DIAGONAL = " << diagonal << endl;
    return 0;
}

File: Course_01/Python/EstruturaSequencial/retangulo.py
import math

base = float(input("Base do retângulo: "))
altura = float(input("Altura do retângulo: "))

area = base * altura
perimetro = 2 * base + 2 * altura
diagonal = math.sqrt(base**2 + altura**2)

print(f"Área = {area:.4f}")
print(f"Perimetro = {perimetro:.4f}")
print(f"Diagonal = {diagonal:.4f}")
